actors/evm/src/interpreter/instructions/ext.rs

```diff
@@ -6,30 +14,74 @@ use {
 
 #[inline]
 pub fn extcodesize<'r, BS: Blockstore, RT: Runtime<BS>>(
-    _state: &mut ExecutionState,
-    _platform: &'r System<'r, BS, RT>,
+    state: &mut ExecutionState,
+    platform: &'r System<'r, BS, RT>,
 ) -> Result<(), StatusCode> {
-    // TODO
-    //  1. call actor::get_actor_code_cid
-    //  2. check that it matches our code CID (it's an EVM actor)
-    //  3. call GetEvmBytecode method, returns the CID of the EVM bytecode block
-    //  4. open the block
-    //  5. return the length
-    todo!()
+    let addr = state.stack.pop();
+    let len = get_evm_bytecode_cid(platform.rt, addr)
```

Comment:
add a TODO about efficiency and `block_stat` here?

Reply:
or maybe inside the impl, at any rate it has to be somewhere so that we remember to fix it.